src/content-helper/editor-sidebar/smart-linking/component-settings.tsx

```diff
@@ -141,16 +175,17 @@ export const SmartLinkingSettings = ( {
 		// If there isn't a selected block, move the focus to the
 		// "All Blocks" button and set the hint to the user.
 		if ( ! selectedBlock && applyTo !== ApplyToOptions.All ) {
-			// If the button changing animation is running, wait for it to finish.
-			if ( animationIsRunning ) {
-				setTimeout( moveButtonAndShowHint, 500 );
-			} else {
-				moveButtonAndShowHint();
-			}
+			moveButtonAndShowHint( __( 'Select a block to apply Smart Links.', 'wp-parsely' ) );
+		}
+
+		// If the selected block is not allowed, move the focus to the
+		// "All Blocks" button and set the hint to the user.
+		if ( selectedBlock && applyTo !== ApplyToOptions.All && ! ALLOWED_BLOCKS.includes( selectedBlock.name ) ) {
+			moveButtonAndShowHint( __( 'This individual block is not supported for Smart Links.', 'wp-parsely' ) );
```

Comment:
The problem isn't the individual block here, but the block type, correct?
In this case, we could update the message to something like "This block type isn't supported by Smart Linking."

Reply:
We could even include the block type name if we wanted, though this isn't a requirement.

Reply:
Thanks, that a pretty good idea! I did that, and I think it makes it easier to understand the message.
![image](https://github.com/user-attachments/assets/1956e2ea-ea74-4537-8413-56ba74015088)